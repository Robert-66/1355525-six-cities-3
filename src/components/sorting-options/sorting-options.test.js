import React from 'react';
import renderer from 'react-test-renderer';
import SortingOptions from './sorting-options';

const options = [`Popular`, `Price: low to high`, `Price: high to low`, `Top rated first`];

it(`SortingOptions is rendered correctly`, () => {
  const tree = renderer
    .create(
        <SortingOptions
          options={options}
          activeOptionIndex={0}
          isOpen={false}
          onClickOption={() => {}}
          onClickToggle={() => {}}
        />
    )
    .toJSON();

  expect(tree).toMatchSnapshot();
});
