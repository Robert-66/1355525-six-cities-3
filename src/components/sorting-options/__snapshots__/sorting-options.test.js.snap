// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SortingOptions is rendered correctly 1`] = `
Array [
  <span
    className="places__sorting-caption"
  >
    Sort by
  </span>,
  "Â ",
  <span
    className="places__sorting-type"
    onClick={[Function]}
    tabIndex="0"
  >
    Popular
    <svg
      className="places__sorting-arrow"
      height="4"
      width="7"
    >
      <use
        xlinkHref="#icon-arrow-select"
      />
    </svg>
  </span>,
  <ul
    className="places__options places__options--custom"
  >
    <li
      className="places__option places__option--active"
      onClick={[Function]}
    >
      Popular
    </li>
    <li
      className="places__option"
      onClick={[Function]}
    >
      Price: low to high
    </li>
    <li
      className="places__option"
      onClick={[Function]}
    >
      Price: high to low
    </li>
    <li
      className="places__option"
      onClick={[Function]}
    >
      Top rated first
    </li>
  </ul>,
]
`;
